### PR TITLE
Adding an overview document for the /dotnet/ai section.

### PR DESCRIPTION
**Pull request by jmatthiesen**
## Summary

Adding a new overview article for the .NET AI documentation.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/ai/get-started/dotnet-ai-overview.md](https://github.com/dotnet/docs/blob/8fc8d2681d2af921e82e73ff1489f5e398062cee/docs/ai/get-started/dotnet-ai-overview.md) | [Develop .NET applications with AI features](https://review.learn.microsoft.com/en-us/dotnet/ai/get-started/dotnet-ai-overview?branch=pr-en-us-40733) |

### CODE REVIEW

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Recommended learning path
+
+We recommend the following sequence of tutorials and articles for an introduction to developing applications with AI and .NET:
+
+1. Follow a tutorial to learn about the basics of AI:
+
+   |Scenario  |Tutorial  |
+   |----------|----------|
+   | Creating a chat application | [Build an Azure AI chat app with .NET](../quickstarts/get-started-azure-openai.md)|
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:41]:
```suggestion
   | Create a chat application | [Build an Azure AI chat app with .NET](../quickstarts/get-started-azure-openai.md)|
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   | Summarizing text | [Summarize text using Azure AI chat app with .NET](../quickstarts/quickstart-openai-summarize-text.md) |
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:42]:
```suggestion
   | Summarize text | [Summarize text using Azure AI chat app with .NET](../quickstarts/quickstart-openai-summarize-text.md) |
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+   | Chatting with your data     | [Get insight about your data from an .NET Azure AI chat app](../quickstarts/quickstart-ai-chat-with-data.md) |
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:43]:
```suggestion
   | Chat with your data     | [Get insight about your data from an .NET Azure AI chat app](../quickstarts/quickstart-ai-chat-with-data.md) |
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+   | Call .NET functions with AI | [Extend Azure AI using Tools and execute a local Function with .NET](../quickstarts/quickstart-azure-openai-tool.md) |
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:44]:
```suggestion
   | Call .NET functions with AI | [Extend Azure AI using tools and execute a local function with .NET](../quickstarts/quickstart-azure-openai-tool.md) |
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+   | Generating images | [Generate images using Azure AI with .NET](../quickstarts/quickstart-openai-generate-images.md) |
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:45]:
```suggestion
   | Generate images | [Generate images using Azure AI with .NET](../quickstarts/quickstart-openai-generate-images.md) |
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+   | Training your own model |[ML.NET Tutorial](https://dotnet.microsoft.com/learn/ml-dotnet/get-started-tutorial/intro) |
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:46]:
```suggestion
   | Train your own model |[ML.NET Tutorial](https://dotnet.microsoft.com/learn/ml-dotnet/get-started-tutorial/intro) |
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Browse the Table of Contents to learn more about the core concepts, starting with [How generative AI and LLMs work](../conceptual/how-genai-and-llms-work.md)
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:48]:
```suggestion
- Browse the table of contents to learn more about the core concepts, starting with [How generative AI and LLMs work](../conceptual/how-genai-and-llms-work.md)
```

File: docs/ai/get-started/dotnet-ai-overview.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+## Next Steps
```
Comment (gewarren) [inline docs/ai/get-started/dotnet-ai-overview.md:50]:
```suggestion
## Next steps
```

**Review by gewarren — APPROVED** [review]
Should the overview also have an entry in the landing page (index.yml)?

**Review by codemillmatt — APPROVED** [review]
I like the article thank you @jmatthiesen! And I really appreciate everything you have done for this project @gewarren!

**Comment by jmatthiesen** [comment]
Thanks for the feedback @gewarren! Your edits were great.

**Pull request merged**